src/Microsoft.AspNetCore.Razor.Evolution/Microsoft.AspNetCore.Razor.Evolution.csproj

```diff
@@ -4,7 +4,7 @@
 
   <PropertyGroup>
     <Description>Razor is a markup syntax for adding server-side logic to web pages. This package contains the Razor parser and code generation infrastructure.</Description>
-    <TargetFrameworks>net451;netstandard1.3</TargetFrameworks>
+    <TargetFrameworks>net451;net46;netstandard1.3</TargetFrameworks>
```

Comment:
That would require changing Mvc. I'd rather not randomize @NTaylorMullen's super crazy https://github.com/aspnet/Mvc/pull/5749 PR for the time being. I'll file a follow up to remove the tfm once he's done merging it.